src/Microsoft.Security.Utilities.Core/DataClassification/DAT101_001_Ip4.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class Ip4 : RegexPattern
```

Comment:
>Ip4 [](http://example.com/codeflow?start=17&length=3)

NIT, Ip4 does not meet .net casing standard, where IP4 would.

Maybe we could use `IPv4`? This is the term I've mostly seen, it's the title for this. e.g., [https://en.wikipedia.org/wiki/IPv4](https://en.wikipedia.org/wiki/IPv4)